app/assets/javascripts/autocomplete.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
```

Comment:
Would be great to either convert this to React or Vue.js code for now -- I can see this class simplifying a lot and living alongside the markup (potentially CSS code too). 

That can totally come at a later time.